subsys/bluetooth/mesh/dfu_srv.c

```diff
@@ -451,8 +452,18 @@ static int dfu_srv_init(const struct bt_mesh_model *mod)
 		return -EINVAL;
 	}
 
-	if (IS_ENABLED(CONFIG_BT_MESH_MODEL_EXTENSIONS)) {
-		bt_mesh_model_extend(mod, srv->blob.mod);
+	const struct bt_mesh_model *blob_srv =
+		bt_mesh_model_find(bt_mesh_model_elem(mod), BT_MESH_MODEL_ID_BLOB_SRV);
+
+	if (blob_srv == NULL) {
+		LOG_ERR("Firmware Update Server cannot extend BLOB Transfer Server");
```

Comment:
```suggestion
		LOG_ERR("Missing BLOB srv");
```
Long debug string are not very useful and eat up the flash during development stage leading to misconceptions about footprint.